dissect/ffs/ffs.py

```diff
@@ -79,27 +79,32 @@ def cylinder_groups(self):
             yield self.cylinder_group(num)
 
     @lru_cache(4096)
-    def inode(self, inum, name=None, filetype=None):
-        return INode(self, inum, name, filetype)
+    def inode(self, inum, name=None, filetype=None, parent=None):
+        return INode(self, inum, name, filetype, parent=parent)
 
     def get(self, path, node=None):
         if isinstance(path, int):
             return self.inode(path)
 
-        node = node or self.root
+        path = path.replace("\\", "/")
```

Comment:
This might actually not be the best idea since `\` is a valid path character I believe?

Reply:
I copied this from XFS/ExtFS, but it looked suspicious to me as well, to be honest...

Reply:
I think that'd be best, yes.

Reply:
Yes, good point, otherwise we keep adding changes.

Reply:
https://jira.nccgroup.com/jira/browse/DIS-1664